ncbi_genome_download/core.py

```diff
@@ -192,7 +199,7 @@ def config_download(config):
                 return 1
 
         if config.metadata_table:
-            with codecs.open(config.metadata_table, mode='w', encoding='utf-8') as handle:
+            with open(config.metadata_table, 'wt') as handle:
```

Comment:
Why is this needed? With this change you're basically rolling back 8c844ab0 and re-opening issue #78 for Python 2 installs.

Reply:
Sorry, looks like that commit came after I forked and I didn't pull/rebase to the latest before submitting the PR. Fixed.